ee/clickhouse/queries/funnels/base.py

```diff
@@ -82,6 +88,24 @@ def _update_filters(self):
         if self._filter.breakdown and not self._filter.breakdown_type:
             data.update({"breakdown_type": "event"})
 
+        # the API accepts either:
+        #   a string (single breakdown)
+        #   a list of numbers (one or more cohorts)
+        #   a list of strings (multiple breakdown)
+        # if the breakdown is a string, box it as a list to reduce paths through the code
+        #
+        # The code below allows you to ensure breakdown is always an array
+        # without it affecting the multiple areas of the code outside of funnels that use breakdown
+        # really it should be boxed at ingestion and be a list everywhere
+
+        if isinstance(self._filter.breakdown, str) and self._filter.breakdown_type in ["person", "event", None]:
+            boxed_breakdown: List[Union[str, int]] = box_value(self._filter.breakdown)
+            data.update({"breakdown": boxed_breakdown})
+
+        if isinstance(self._filter.funnel_step_breakdown, str):
+            if self._filter.funnel_step_breakdown.startswith("["):  # naive list as string detection
```

Comment:
Is it worth, rather than overloading the property names, separating the new functionality out to, say, `funnel_step_breakdowns`, and perhaps do the same for the response format. It should reduce the risks, and make it easier to remove the old logic once we've migrated.

Reply:
Yes, absolutely. I think the API needs to change. My plan is to add multi-property breakdown to trends too. To validate that the change would be appropriate there as well.